tool/site/client/tests/pages/withdrawPage.spec.tsx

```diff
@@ -120,14 +124,14 @@ describe("Withdraw Page", () => {
         }
       })
 
-      renderWithContext(<WithdrawPage prescriptionId={prescriptionId}/>, context).container
-      await waitFor(() => screen.getByText("Withdraw prescription"))
+      container = renderWithContext(<WithdrawPage prescriptionId={prescriptionId}/>, context).container
+      await waitFor(() => screen.getByText(`Withdrawing Dispense: ${dispenseNotificationId}`))
       userEvent.click(screen.getByText("Withdraw"))
       await waitFor(() => screen.getByText("Withdraw Result"))
     })
 
     it("should display one dispense notifications", () => {
-      expect(screen.getAllByText("76d1cc0b-bd64-4fad-a513-4de0f2ae7014")).toHaveLength(1)
+      expect(container.getElementsByClassName("nhsuk-expander").length).toBe(5)
```

Comment:
'it should display event one' & 'it should not display event two'

Reply:
done